pywt/tests/test__pywt.py

```diff
@@ -7,6 +7,8 @@
 
 import pywt
 
+rstate = np.random.RandomState(1234)
```

Comment:
Tests are not guaranteed to run in a fixed order, so you should create a ``RandomState`` object per test instead of per file.